cloud-optimized-geotiffs/cogs-details.qmd

```diff
@@ -33,9 +34,9 @@ Read performance can be greatly impacted by the choice of projection and the par
 * The optimum size of data at which splitting across files improves performance as a multi-file dataset instead of a single file.
 * When to recommend particular internal tile sizes
 * Compression impacts on http transfer rates.
```

Comment:
What is this supposed to mean? Does this mean "full time to load and parse an image in a browser"? Or "Compression speed for streaming compression"? Except maybe for streaming compression/decompression on a server, the only impact on http transfer rates would be the size of the file...?

Reply:
@abarciauskas-bgse did you write that line originally?

Reply:
@kylebarron I think this goes back to my question about how/if the internal chunks are compressed, and if having an internally compressed file means that file transfers are faster even if doing partial reads.

Reply:
each blocks are compressed independently (you can easily see the difference when you change the internal tile size).

Reply:
original file: https://docs.google.com/document/d/1rBRr4xcz2NH3JXS0Y8wNEqvd2-pzVw23_p2NV3jKnlM/edit

Reply:
> I think this goes back to my question about how/if the internal chunks are compressed

I added some text in the compression section mentioning that although maximum compression can be slower to decompress, if your network is slow, the smaller size may outweigh the time to decompress. Not sure if this is related to the same point as above

Reply:
I think it is, and I've never actually seen a compression comparison where network was added as a factor. With local filesystem Deflate 6 was the optimum and the space savings of 9 resulted in significantly slower read times.